lidarr/sonarr_test.go

```diff
@@ -0,0 +1,19 @@
+package lidarr_test
```

Comment:
change filename to lidarr_test.go

Reply:
haha nice catch!

Reply:
I actually changed GetInto in another branch, and it's going to ruin all these tests. I need to merge the branches and fix this. I appreciate you taking a look, but please check back in a couple days after I get things squared away. I have a lot of pending stuff going on in this library right now.